python/ray/workflow/event_providers/http_event_provider.py

```diff
@@ -0,0 +1,82 @@
+import ray
```

Comment:
let's just put it into workflow's folder for now. We might need to put events related things into events folder in the future.